modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,198 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
 In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
 
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift Container Platform cluster that is running on ARM hardware.
++
+include::snippets/technology-preview.adoc[]
 
-With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift Container Platform cluster that is running on ARM hardware.
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
+// (link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
 
-include::snippets/technology-preview.adoc[]
+* Previously, as a developer, you could run only one pipeline run from your `.tekton` directory for each Git event, such as a pull-request. With this update, you can run all the pipeline runs in your `.tekton` directory that match specified events. The web console displays the status and reports of the runs. If you are using WebVCS, the reports include comments.
```

Comment:
```suggestion
* Previously, as a developer, you could run only one pipeline run from your `.tekton` directory for each Git event, such as a pull-request. With this update, you can run all the pipeline runs in your `.tekton` directory that match specified events. The web console displays the status and reports of the runs. If you are using WebVCS, the reports include comments.
```
`run one pipeline run` -- is this correct?

Reply:
Thanks. I revised this to "having a pipeline run."